toolchain/sem_ir/inst_namer.cpp

```diff
@@ -416,6 +416,13 @@ auto InstNamer::CollectNamesInBlock(ScopeId scope_id,
             sem_ir_.entity_names().Get(inst.entity_name_id).name_id);
         continue;
       }
+      case BindingPattern::Kind:
+      case SymbolicBindingPattern::Kind: {
+        auto inst = untyped_inst.As<AnyBindingPattern>();
+        add_inst_name_id(
+            sem_ir_.entity_names().Get(inst.entity_name_id).name_id, ".patt");
```

Comment:
Note: the "`.patt`" never appears in the textual IR output. The only effect of omitting that parameter here is that the instruction numbering changes slightly. I can't tell whether any of that is working as intended.